src/components/data_submission/consent_group/ConsentGroupForm.js

```diff
@@ -12,38 +12,42 @@ export const ConsentGroupForm = (props) => {
     updateNihInstitutionalCertificationFile,
     deleteConsentGroup,
     disableDelete,
+    consentGroupsState,
+    studyEditMode
   } = props;
 
   const [consentGroup, setConsentGroup] = useState({
```

Comment:
```suggestion
  const curConsentGroup = consentGroupsState[idx]?.consentGroup; // not sure if ? is needed.

  const [consentGroup, setConsentGroup] = useState({
```

It might be useful to pull this out beforehand so you can use it for all the properties below.